sky/tools/gn

```diff
@@ -44,6 +48,7 @@ def to_gn_args(args):
 
     gn_args['is_debug'] = args.debug
     gn_args['is_clang'] = args.clang and args.target_os not in ['android', 'fnl']
+    gn_args['flutter_develop_mode'] = args.develop
```

Comment:
So the defaults are that develop mode is false and dart product mode is false?  Does that combination compile?